site/docs/v1/tech/apis/_two-factor-disable-request-body.adoc

```diff
@@ -0,0 +1,16 @@
+==== Request Body
+
+[.api]
+[field]#applicationId# [type]#[UUID]# [optional]#Optional#::
+The Id of the application. If valid, the email template configured in the Application settings will be used, if present. If not present, the email template configured in the Tenant settings will be used. In either case, the corresponding Application object will be available to the template.
+To learn more about overriding email templates, read our link:/docs/v1/tech/guides/configuring-application-specific-email-templates[Application Specific Email Templates] guide.
+
+[field]#code# [type]#[String]# [required]#Required#::
+A valid Multi-Factor verification code. This value should be provided by the User to verify they are able to produce codes.
```

Comment:
The name of the endpoint is `Disable Multi-Factor Authentication With a UserId`. Although it's the Two Factor API, most of the language on the page uses Multi-Factor > Two Factor. I copy and pasted this from the request parameters so I will be sure to clean make it apply to TOTP, Email, and SMS.

Reply:
I can change this to favor `two-factor` over `multi-factor but I wanted to confirm the change before I did.

Reply:
After reviewing the Multi-Factor/Two-Factor API, IMO, I think it makes more sense to keep Multi-Factor. When you enable multi-factor, you can select 1 of 3 values: authenticator, email, SMS. Since multiple methods can be enabled, I think it makes more sense to keep the 73 mentions of Multi-Factor over two-factor.

Reply:
@robotdan - I'm sticking with `Multi-Factor` over `multi-factor` - otherwise, I would have to update all other fields. Would you rather me update all `Mutli-Factor` -> `multi-factor` or continue to use `Multi-Factor`?

Reply:
made some edits to use `multi-factor`